recipes/yacht/meta.yaml

```diff
@@ -1,24 +1,33 @@
-{% set version = "1.2.3" %}
+{% set version = "1.3.0" %}
 
 package:
   name: yacht
   version: {{ version }}
 
 source:
   url: https://github.com/KoslickiLab/YACHT/releases/download/v{{ version  }}/yacht-{{ version  }}.tar.gz
-  sha256: 93adb23ad4f143d48c9f0ea661fe2283ff42e9c51e073a05460596ecabc65214
+  sha256: 04a1fabffb6767619f84e08f74126ba2be02d146169bd76980ee80ffe3cb0039
 
 build:
   number: 0
-  noarch: python
-  skip: True  # [win or osx]
-  script: "{{ PYTHON  }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv"
+  skip: True  # [osx]
+  script: "{{ PYTHON }} -m pip install . --no-deps --no-build-isolation --no-cache-dir -vvv"
   run_exports:
-    - {{ pin_subpackage('yacht', max_pin="x") }}
+    - {{ pin_subpackage('yacht') }}
 
 requirements:
+  build:
+    - {{ compiler('cxx') }}       # Adds platform-specific C++ compiler (g++, clang, MSVC)
+    - gxx_linux-64           # Ensures g++ is available on Linux
+    - zlib                     # zlib for compression
```

Comment:
Only 'make' should stay in 'build'. All others are not build tools but libraries and should be moved to 'host'

Reply:
Thanks @martin-g. I have removed others but only leave `make` in 'build'. Let's see if it works.

Reply:
Hi @martin-g, it seems like g++ is not provided in the server if I removed.

```
20:10:20 BIOCONDA INFO (OUT)   make: g++: No such file or directory
```
Any idea on how to fix this? Thanks in advance!